src/NUnitEngine/nunit.engine/Internal/SettingsStore.cs

```diff
@@ -89,7 +89,8 @@ public void LoadSettings()
             }
             catch (Exception ex)
             {
-                throw new ApplicationException("Error loading settings file", ex);
+                string msg = string.Format("Error loading settings {0}.", _settingsFile);
+                throw new ApplicationException(msg, ex);
```

Comment:
Good idea, I will update. Do we show the inner exception in that case? If not, I will add the message from the inner exception. When I was testing, it told me exactly what was wrong with the file.